spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,12 +195,14 @@
 
     context "when a new workflow has been chosen" do
       it "gives the managers workflow roles" do
+        expect(form).not_to receive(:update_permission_template)
```

Comment:
Now that the workflow migration patch (https://github.com/projecthydra-labs/hyrax/pull/548) has been merged, there is a way to create a failing test. Prior to this migration, this spec would not have caused a failing test because the issue was caused by an incompatibility between the permission template in Hyrax vs the one in Hyku (which had gotten migrated).  This test was most likely another victim of the merge conflict which caused the problematic code to remain in the first place. But regardless, the issue is now resolved.